File: typescript/8kyu/Basic_Mathematical_Operations.ts
export const basicOp = (
  operation: string,
  value1: number,
  value2: number
): number =>
  operation === "+"
    ? value1 + value2
    : operation === "-"
    ? value1 - value2
    : operation === "*"
    ? value1 * value2
    : operation === "/"
    ? value1 / value2
    : 0;

File: rust/src/8kyu/Basic_Mathematical_Operations.rs
fn basic_op(operator: char, value1: i32, value2: i32) -> i32 {
    if operator == '+' {
        value1 + value2
    } else if operator == '-' {
        value1 - value2
    } else if operator == '*' {
        value1 * value2
    } else if operator == '/' {
        value1 / value2
    } else {
        0
    }
}
